src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -222,6 +228,13 @@ public void configure() throws Exception {
                     exchange.getIn().setHeader(USERNAME, userName);
                 });
 
+        from("direct:add-userAccount-header")
```

Comment:
I haven't seen any test for this route similar to **MainRouteBuilder_DirectAddUsernameHeaderTest**

Reply:
removed route after refactoring